packages/react-ui-components/src/components/SchemaForm/Templates/TitleFieldTemplate/TitleFieldTemplate.tsx

```diff
@@ -0,0 +1,20 @@
+import { TitleFieldProps } from '@rjsf/utils';
+import { get, isEmpty } from 'lodash-es';
+import React from 'react';
+import { KvInfoLabel } from '../../../stencil-generated';
+import styles from './TitleFieldTemplate.module.scss';
+
+const TitleFieldTemplate = ({ title, uiSchema, required }: TitleFieldProps): any => {
+	const titleToShow = get(uiSchema, ['ui:title'], title);
+	return (
+		!isEmpty(titleToShow) &&
+		titleToShow !== ' ' && (
```

Comment:
Why are you checking this condition?

Reply:
Because if we set the displayLabel to empty string to not display the label the RJSF lib will auto generate a label.

Reply:
Really? Ok, then 😄 

Would it make sense to create a util function only to perform this check? It's not because it's a complex logic but at first look seems a little strange.